node/pkg/processor/broadcast.go

```diff
@@ -23,15 +23,18 @@ var (
 		})
 )
 
-func (p *Processor) broadcastSignature(v *vaa.VAA, signature []byte, txhash []byte) {
-	digest := v.SigningMsg()
-
+func (p *Processor) broadcastSignature(
+	o Observation,
+	digest ethcommon.Hash,
```

Comment:
Is there a reason you're not using the digest available on the Observation interface (it's already available as hex, why not as binary digest)?

Reply:
Good point.  I didn't realize that HexDigest was just the hex of SigningMsg.  I've updated the Observation interface to use SigningMsg instead and removed this parameter from broadcastSignature.